### PR TITLE
voting period is open when voting dates are not set

### PR DESCRIPTION
**Pull request by differentreality**
Fixes bug from #987

### CODE REVIEW

File: spec/models/program_spec.rb [hunk 1]
```diff
@@ -90,7 +90,7 @@
 
   describe 'voting_period?' do
     it 'retuns false when voting dates are not set' do
-      expect(program.voting_period?).to eq false
+      expect(program.voting_period?).to eq true
```
Comment (ChrisBr) [inline spec/models/program_spec.rb]: Huh?

Comment (differentreality) [inline spec/models/program_spec.rb]: Until we did not have voting dates, the voting period was always on (technically even after the conference). I want to maintain that behavior for past conferences.

Comment (ChrisBr) [inline spec/models/program_spec.rb]: Ok, but the test is then not doing what it does say ;)

Comment (differentreality) [inline spec/models/program_spec.rb]: Yeah that was unfortunate :)

**Comment by differentreality** [comment]
Can someone review this?

**Comment by differentreality** [comment]
@ChrisBr @hennevogel can you review this? (Or anyone else?)

File: spec/models/program_spec.rb [hunk 2]
```diff
@@ -90,8 +90,8 @@
   end
 
   describe 'voting_period?' do
-    it 'retuns false when voting dates are not set' do
-      expect(program.voting_period?).to eq false
+    it 'retuns true when voting dates are not set' do
```
Comment (ChrisBr) [inline spec/models/program_spec.rb]: Why is that necessary now?

Comment (differentreality) [inline spec/models/program_spec.rb]: Compatibility with pre-existing conferences that do not have voting dates, and essentially no change in workflow for ppl already used to that logic. That practically means voting is still possible after the conference, which might not make much sense, but that's what happens now in our existing conferences.

Comment (ChrisBr) [inline spec/models/program_spec.rb]: Ah ok!

File: app/models/program.rb [hunk 3]
```diff
@@ -72,16 +72,16 @@ def selected_event_schedules
...
 
   ##
   # Checks if we are still in voting period
   # ====Returns
-  # * +true+ -> If the voting period is not over yet
+  # * +true+ -> If the voting period is not over yet (or if the voting dates are not set)
   # * +false+ -> If the voting period is over
   def voting_period?
-    return false unless voting_start_date && voting_end_date
+    return true unless voting_start_date && voting_end_date
```
Comment (ChrisBr) [inline app/models/program.rb]: Only thing I'm now concerned is that the function name does clearly not cover what it is doing :(
